components/shared/script/lib.rs

```diff
@@ -808,6 +808,15 @@ pub enum WebDriverCommandMsg {
         Option<Rect<f32, CSSPixel>>,
         IpcSender<Option<Image>>,
     ),
+    /// Create a new tab that loads about:blank. The constellation will use
+    /// the provided channels to return the top level browsing cntext id
+    /// associated with the new tab, and a notification when the initial
+    /// load is complete.
+    NewTab(IpcSender<TopLevelBrowsingContextId>, IpcSender<LoadStatus>),
+    /// Close the webview associated with the provided id.
+    CloseWindow(TopLevelBrowsingContextId),
+    /// Focus the webview associated with the provided id.
+    FocusWebView(TopLevelBrowsingContextId),
```

Comment:
I feel like these variants should share the same term here. Could they all be `WebView` like [`ConstellationMsg`](https://github.com/servo/servo/blob/bc2def0582ff374d72633f2a06b47842483bac47/components/shared/compositing/constellation_msg.rs#L58) does?

Reply:
Done.